src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -164,6 +164,10 @@ export default class OrganizationDiscover extends React.Component {
 
     const shouldDisplayResult = resultManager.shouldDisplayResult();
 
+    const projects = organization.projects.filter(
+      project => (project.isMember ? project : null)
```

Comment:
I think this line should be `project => project.isMember` (filter callback should be a boolean)

Reply:
👍 thanks!!